1.download-data/scripts/download_data.py

```diff
@@ -0,0 +1,30 @@
+import urllib.request
+import shutil
+import os
+
+
+def downloadData(
+    locationData: str = "temp.zip",
+    url: str = r"https://ndownloader.figshare.com/files/18501824?private_link=f41918598b1ff5116825",
```

Comment:
```suggestion
    url: str = r"https://ndownloader.figshare.com/files/18501824",
```

Updating to public URL

Reply:
Removed the default values in the function. So we only have to maintain the ones from the parser